### PR TITLE
Apache Jena EARL report - 2021-04-09

### PR DESCRIPTION
**Pull request by afs**

### CODE REVIEW

File: reports/jena-earl.ttl [hunk 1]
```diff
@@ -0,0 +1,2058 @@
...
+        rdf:type         doap:Project ;
+        dc:creator       _:b0 ;
+        dc:title         "Apache Jena" ;
+        doap:homepage    "http://jena.apache.org/" ;
+        doap:maintainer  _:b0 ;
+        doap:developer   _:b0 ;
+        doap:name        "Apache Jena" ;
+        doap:release     [ rdf:type      doap:Version ;
+                           doap:created  "2021-04-01"^^xsd:date ;
+                           doap:name     "4.0.0"
```
Comment (gkellogg) [inline reports/jena-earl.ttl:2054]:
```suggestion
                            doap:revision     "4.0.0"
```

**Comment by gkellogg** [comment]
Looks good other then Version having a "revision" instead of a "name" (at least, that's how the reporting software interprets the data).

The "Apache Jena Community" should not try to link to a blank node, which is a small correction needed in the Haml template I'll take care of after the merge.

**Comment by afs** [comment]
Fixed and tidied up.

**Comment by afs** [comment]
Apologies - the version was wrong (the report generator is newer to conform with the submission requirements) than the system under test). There should not be a "SNAPSHOT". Fixed in the reports area.

**Comment by gkellogg** [comment]
Do you want to submit an update, or patch in place?

**Comment by afs** [comment]
I fixed it in-place (commit 2b791b06) and reran the report generator - a test of updating a report, not just adding one.

It updated and all went fine.

**Pull request merged**